src/index.js

```diff
@@ -43,6 +42,9 @@ class Node extends EventEmitter {
       // If muxer exists, we can use Identify
       this.swarm.connection.reuse()
 
+      // If muxer exists, we can use Relay for listening/dialing
+      this.swarm.connection.enableCircuitRelay(_options.relay)
```

Comment:
This should be enabled by an option and not by default

Reply:
this is being handled inside the `enableCircuitRelay` - https://github.com/libp2p/js-libp2p-swarm/blob/master/src/connection.js#L100.